### PR TITLE
Release 2.2.0

### PR DESCRIPTION
**Pull request by lockw1n**
- Developer(s): @aleksey-kotsuba

- Ticket: https://spryker.atlassian.net/browse/ECO-2670

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   ArvatoRss             | minor                 |                       |

#### Release Notes

Added `CommunicationToken` into `storeOrder` API call on module level.

-----------------------------------------

##### Change log

Improvements:
 - Added `communication_token` field into `spy_arvato_rss_api_call_log` table.
 - Added checkout save order plugin to update api log with order reference.
 - Communication token has been added to storeOrder request.
 - Added optional `ArvatoRssReaderInterface` dependency into `StoreOrderRequestMapper ` to retrieve communication token from DB.

### CODE REVIEW

File: tests/SprykerEcoTest/Zed/ArvatoRss/_support/_generated/ArvatoRssZedTesterActions.php [hunk 1]
```diff
@@ -1,19 +1,10 @@
-<?php  //[STAMP] 2ab9889b9623cb01b27a96cd0a920d95
```
Comment (gechetspr) [inline tests/SprykerEcoTest/Zed/ArvatoRss/_support/_generated/ArvatoRssZedTesterActions.php:1]: Shouldn't this be under gitignore?

**Review by gechetspr — APPROVED** [review]
Please check, why generated files are not under gitignore, but the rest looks ok

**Review by alex-galych — APPROVED** [review]

File: tests/SprykerEcoTest/Zed/ArvatoRss/_support/_generated/ArvatoRssZedTesterActions.php (hunk 1 above)
Comment (lockw1n) [reply tests/SprykerEcoTest/Zed/ArvatoRss/_support/_generated/ArvatoRssZedTesterActions.php:1]: I'm not sure why but we have this file under version control in all our modules.

Comment (gechetspr) [reply tests/SprykerEcoTest/Zed/ArvatoRss/_support/_generated/ArvatoRssZedTesterActions.php:1]: Ok. It's not a blocker

**Pull request merged**